src/Table.js

```diff
@@ -10,32 +10,35 @@ import {
 } from './utils/bootstrapUtils';
```

Comment:
We just merged some changes to how we do styling, can you take a look a the Button component as an example and use `createBootstrap` instead of any utils from bootstrapUtils? thanks!